src/pegout/components/PegoutOption.vue

```diff
@@ -1,20 +1,20 @@
 <template>
   <v-card :ripple="false" rounded="lg" flat variant="outlined"
-    @click="selectOption" height="450px" :disabled="flyoverNotAvailable"
+    @click="selectOption" height="400px" :disabled="flyoverNotAvailable"
```

Comment:
Can we change it to use columns? It could be broken in small screens.

Reply:
It can be done, but it would make that both options have different size, like this:
![Screenshot 2024-11-21 at 1 20 09 PM](https://github.com/user-attachments/assets/361fe246-7966-4978-9281-108a3a835ae4)
![Screenshot 2024-11-21 at 1 20 39 PM](https://github.com/user-attachments/assets/f3ac910e-d19a-4f8b-b3b4-682c0a47c54e)

It is ok?

Reply:
Is decided to keep the change `height="400px"` and test it on different screen sizes.